### PR TITLE
Bankers Algorithm, Travelling Salesman Problem, Sliding Window and Wave Sort algorithm Splay tree algorithm

### PR DESCRIPTION
**Pull request by mvpamansingh**
## 🛠️ Issue (151)

Issue no #151 
Issue no #225
Issue no #170 
Issue no #175
Issue no #231 c and java
## ✔️ Check List (Check all the applicable boxes) 

- [ ✔️] My code follows the code style of this project.
- [✔️ ] This PR does not contain plagiarized content.
- [✔️ ] The title of my pull request is a short description of the requested changes.

## 📷 Screenshots

### CODE REVIEW

**Review by Kumar-laxmi — CHANGES_REQUESTED** [review]
Changes required:
- Update the title to include "Wave Sort"
- In description please include the issue number of Wave Sort

**Comment by Kumar-laxmi** [comment]
@mvpamansingh Please resolve the **CodeFactor** issue

**Comment by mvpamansingh** [comment]
@Kumar-laxmi I have fix the code factor for wave sort algorithm

**Review by Kumar-laxmi — CHANGES_REQUESTED** [review]
Changes to be done:
- Remove folder ```.vscode```
- Update the PR title to : "Bankers Algorithm, Travelling Salesman Problem, Sliding Window and Wave Sort algorithm"

File: Java/Sliding Window/SlidingWindow.java [hunk 1]
```diff
@@ -0,0 +1,31 @@
+public class SlidingWindow {
```
Comment (Kumar-laxmi) [inline Java/Sliding Window/SlidingWindow.java:1]: Implement all problems in Sliding Window problems in Sliding Window Folder for **bonus score**

**Review by 072arushi — APPROVED** [review]
Approved

**Review by zugzwang03 — COMMENTED** [review]
Adding comment lines explaining your approach would have been better

**Review by siddhi-244 — CHANGES_REQUESTED** [review]
Can you remove those extra files committed which is the .vscode folder which is really not needed!

**Review by Arun9739 — CHANGES_REQUESTED** [review]
Remove the .vscode foder. It is not necessary for the code.

**Comment by mvpamansingh** [comment]
@Kumar-laxmi @Arun9739 @siddhi-244 I have removed the vs folder, please check the pr

**Review by siddhi-244 — CHANGES_REQUESTED** [review]
Add some comments to explain the code

**Review by Arun9739 — APPROVED** [review]

**Comment by mvpamansingh** [comment]
@siddhi-244 I have added comments ,please review it
@Kumar-laxmi changed the pr title please review it

**Review by zugzwang03 — APPROVED** [review]

**Review by siddhi-244 — APPROVED** [review]

**Comment by mvpamansingh** [comment]
@Kumar-laxmi I have updated the pr title, please review it..

File: Java/Sliding Window/SlidingWindow.java (hunk 1 above)
Comment (Kumar-laxmi) [reply Java/Sliding Window/SlidingWindow.java:1]: @mvpamansingh Please implement all the problems of Sliding Window as mentioned in issue

**Review by Kumar-laxmi — CHANGES_REQUESTED** [review]

**Review by Kumar-laxmi — APPROVED** [review]

**Pull request merged**